README.md

```diff
@@ -110,4 +111,4 @@ License
     distributed under the License is distributed on an "AS IS" BASIS,
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
-    limitations under the License.
\ No newline at end of file
+    limitations under the License.
```

Comment:
Avoid adding unintended changes. This change should be reverted.

Reply:
It's maybe because of the link entry in the readme.